CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+- Add Warsaw (europe-central2) Cloud Function Location to Firebase Extension template.
```

Comment:
This already was included in 9.13.0: https://github.com/firebase/firebase-tools/releases - can you remove it?

Reply:
Will do